Libraries/LibJS/Tests/String.prototype.trim.js

```diff
@@ -0,0 +1,15 @@
+function assert(x) { if(!x) throw x; }
+
+try {
+    assert(String.prototype.trim.length === 0)
+    assert("   hello friends  ".trim() === "hello friends");
+    assert("hello friends   ".trim() === "hello friends");
+    assert("   hello friends".trim() === "hello friends");
+    assert("   hello friends".trimStart() === "hello friends");
+    assert("hello friends   ".trimEnd() === "hello friends");
```

Comment:
this should probably also verify that `trimStart` does _not_ remove whitespace from the end and `timeEnd` from the start.